boards/b-l072z-lrwan1/Makefile.include

```diff
@@ -12,5 +12,8 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 export DEBUG_ADAPTER ?= stlink
 export STLINK_VERSION ?= 2-1
 
+# do a reset before halt when starting debugger
+export PRE_DEBUG_RESET_HALT ?= 1
```

Comment:
I think the variable should be declared somewhere more general, then set it to one here. Maybe makefiles/vars.inc.mk ?

Reply:
Well I guess this anyways will be refactored for all the ST boards including a ST-Link debugger, so it's fine for now.

Reply:
> this anyways it will be refactored for all the ST boards embedding a ST-Link debugger

Are you referring to #8475 ?